### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by williamscch**
In this project, I restructured my Awesome books app code. I made it more organized using modules and practiced the ES6 syntax that I learned.
I also used the Luxonjs library to display the current date below the navigation bar.

### CODE REVIEW

File: modules/classbook.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default class Book {
```
Comment (rustam-tolipov) [inline modules/classbook.js:1]: - [x] Well good class based Component. However, in the requirement you are asked to use function based Components. Kindly change class b to function b 😊.

File: modules/classdisplay.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import Storage from './classstorage.js';
+import { title, author } from './variables-fullpage.js';
+
+export default class Display {
```
Comment (rustam-tolipov) [inline modules/classdisplay.js]: - [x] As we mentioned above, function based 😉.

File: modules/classstorage.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+export default class Storage {
```
Comment (rustam-tolipov) [inline modules/classstorage.js]: - [x] The same as above 🔝

File: README.md [hunk 4]
```diff
@@ -64,9 +39,9 @@ Give a ⭐️ if you like this project!
...
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Original repository made with my team [here](https://github.com/DiegoMero/Awesome-Books)
+- Microverse Module 2
+- Project indications [here](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md)
 
 ## 📝 License
```
Comment (rustam-tolipov) [inline README.md:46]: - [x] [OPTIONAL] Well done for implementing the MIT license in your project. Also, do not forget to personalize it as well.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @wdavidcch, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello @wdavidcch👋,

This is an additional review as a result of the second opinion request.

Yes, you're right 👍, in the previous milestone, you used classes, however, in this milestone which is [ES6 modules](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20methods,a%20correct%20way.),  you need to change classes to arrow functions. Kindly read the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) carefully.
 
Kindly implement the changes requested by the previous CR.

Keep rocking 🚀!

**Review by ShahierNashaat — APPROVED** [review]
Hi @wdavidcch , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**